docs/guides/cookies.md

```diff
@@ -37,6 +39,8 @@ import puppeteer from 'puppeteer';
 
 const browser = await puppeteer.launch();
 
+const context = await browser.createBrowserContext();
```

Comment:
same here

Reply:
done in 1c34f0c3042dc3060d54ce5d64d6c15298f70d88